config/webpack.demo.js

```diff
@@ -0,0 +1,182 @@
+const helpers = require('./helpers');
+const webpack = require('webpack');
+const path = require('path');
+const sass = require('./sass');
```

Comment:
This is an unused variable - see comment below and does not indicate any technology choice.